docs/migrating-vue-router.md

```diff
@@ -0,0 +1,24 @@
+# Migrating from vue-router
+
+:white_check_mark: Nested routes mapping  
+:white_check_mark: Dynamic Routing  
+:white_check_mark: Modular, component-based router configuration  
+:white_check_mark: Route params, query, wildcards  
+:white_check_mark: View transition effects powered by Vue.js' transition system **(todo!)**  
+:white_check_mark: Fine-grained navigation control  
+:white_check_mark: Links with automatic active CSS classes **(todo!)**  
+:x: HTML5 history mode or hash mode  
+:white_check_mark: Customizable Scroll Behavior **(todo!)**  
+:white_check_mark: Proper encoding for URLs **(todo?)**  
+
+## Child Routes
+
+Child routes in vue-router have very different behavior depending on if the path starts with "/" or not. In kitbag/router, the behavior is always the same, so add slashes where you want them and leave them off where you don't.
+
+## Route Regex
+
+### Hash History Mode
+
+## Repeatable Params
```

Comment:
I'm curious if this is supported with a regex param. Like couldn't you just write a regex that would accomplish this?

Reply:
we talked about this, you're right that with params that extend beyond the forward slash we can theoretically support repeatable params, but that's going to change and we're not sure we will support that at launch. Keeping this under not supported for now